common/js/polyfills.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable */
```

Comment:
This polyfill script is still a question mark. Because now it's polyfilling the TextEncoder used to check is the password was leaked. It was introduced by #845 

TextEncoder is not available in Edge, IE nor Safari so I included the polyfill from MDN: https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder#Browser_compatibility

Now that #863 is merged, I was thinking to load the polyfill from NPM.

* Can I Use table for the TextEncoder: https://caniuse.com/#feat=textencoder

That also rose the issue of which are the browser we need to support. I'm pretty sure we need to support Edge and Safari but I don't know about IE11? (At work, since we do B2B, we still need it). If we need to support IE11, I still need to figure out how to generate the SHA1 and to replace fetch to XMLHttpRequest.

* Can I Use table for the web cryptography: https://caniuse.com/#feat=cryptography
* Can I use table for fetch https://caniuse.com/#feat=fetch

Reply:
Let's indeed use npm!

This is my viewpoint regarding browser compatibility: https://github.com/solid/solid-auth-client/blob/v2.2.8/.browserslistr, resulting in https://browserl.ist/?q=node+8%2C+%3E+1%25%2C+not+ie+%3C%3D+11%2C+not+operamini+all